mkdocs_click/_docs.py

```diff
@@ -92,7 +92,7 @@ def _make_usage(ctx: click.Context) -> Iterator[str]:
     full_path.reverse()
     usage_snippet = " ".join(full_path) + usage
 
-    yield "Usage:"
+    yield "**Usage:**"
```

Comment:
What do you think about dropping the colon altogether…?

```suggestion
    yield "**Usage**"
```

Reply:
I left the colon as is with no particular reasons. However, I prefer it to be remaining here because it has a better consistency with [the mkdocstrings' behavior](https://github.com/pawamoy/mkdocstrings/blob/de2ef32e9ccfd2fe59177d5e5bc4d3daf4f20587/src/mkdocstrings/templates/python/material/parameters.html#L1), which in turn originates from the [Google style docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings).

That being said, I'd go the way you prefer.

Reply:
@yudai-nkt Alright, no strong opinion there, let's do "keep the colon". :)